### DISCUSSION TITLE
When is the final (public, non beta) version of v5 likely to be released??

### DISCUSSION THREAD

**Discussion by kelvinelove** [ant-design/ant-design-mobile#4071] [Q&A] [answered]
I've been developing a project using v2.x, and I was considering starting to migrate it to v5. 
Is it advisable to wait for the final release, or start preparing the migration now, since the final release (may?) not be much different from the betas?

Also, more importantly, when is the final version likely to drop. I think knowing that will be helpful in my making a decision. Thanx

**Comment by awmleer** [comment, marked as answer]
We plan to release a few rc versions in October, and publish 5.0 stable version offically in December. If it goes well, the release date may be advanced.

Currently, we recommend starting the migration if possible, since the API has been basically stable and there won't have many break changes in the future.

Here is the full roadmap for 5.0: #3924 (But it's in Chinese)
Also, the documentation site is still mainly in Chinese and we're working hard to translate it. Sorry for the inconvenience.
